### DISCUSSION TITLE
Filenames are not split correctly. isModern is undefined.

### DISCUSSION THREAD

**Discussion by vagelis-chantzis** [webpack/webpack#16868] [Q&A]
Hello,

I am facing the above issue. I want to create filenames based on isDev and isModern values, but isModern is shown as undefined. 
Thus, the produced filenames are never constructed as I want them to.

Nuxt 3 application, webpack 5.75.0

My nuxt.config.ts file is this, showing the webpack configuration part:

webpack: {
    filenames: {
      css: ({ isDev, isModern }) =>
        isDev
          ? `[name]${isModern ? ".modern" : ""}.css`
          : appEnv !== "production"
          ? `[name].[contenthash:7]${isModern ? ".modern" : ""}.css`
          : `[contenthash:7]${isModern ? ".modern" : ""}.css`,
    },

**Comment by alexander-akait** [comment]
Sorry, we don't have `isModern` or `isDev` variables, you can use `cross-env NODE_ENV=development webpack --config webpack.config.js` and use `const isDev = typeof process.env.NODE_ENV !== "undefined" && process.env.NODE_ENV === "development"`, also you can use https://webpack.js.org/api/cli/#env to inject any logic in your configurations
